.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
este archivo no debe ser un parte del repositorio. Usa este sitio para hacer un gitignore, y quita los archivos extras del repositorio
https://www.gitignore.io/api/androidstudio

Reply:
I fix it Katy